ansible_ai_connect/ai/api/tests/test_views.py

```diff
@@ -3397,3 +3405,215 @@ def test_feature_not_enabled_yet(self):
         self.client.force_login(user=self.user)
         r = self.client.post(reverse("explanations"), payload)
         self.assertEqual(r.status_code, 404)
+
+
+class TestChatView(WisdomServiceAPITestCaseBase):
```

Comment:
It took a little to understand; but nice tests :smile_cat:

Reply:
Thanks!